tools/make/kube.mk

```diff
@@ -61,21 +61,15 @@ kube-undeploy: kube-uninstall ## Uninstall the Envoy Gateway controller into the
 
 .PHONY: kube-demo
 kube-demo: ## Deploy a demo backend service, gatewayclass, gateway and httproute resource and test the configuration.
-	kubectl apply -f examples/kubernetes/httpbin.yaml
-	kubectl apply -f examples/kubernetes/gatewayclass.yaml
-	kubectl apply -f examples/kubernetes/gateway.yaml
-	kubectl apply -f examples/kubernetes/httproute.yaml
+	kubectl apply -f examples/kubernetes/quickstart.yaml
 	@echo "\nPort forward to the Envoy service using the command below"
 	@echo "kubectl -n envoy-gateway-system port-forward service/envoy-default-eg 8888:8080 &"
 	@echo "\nCurl the app through Envoy proxy using the command below"
 	@echo "curl --verbose --header \"Host: www.example.com\" http://localhost:8888/get\n"
 
 .PHONY: kube-demo-undeploy
 kube-demo-undeploy: ## Uninstall the Kubernetes resources installed from the `make kube-demo` command.
-	kubectl delete -f examples/kubernetes/httproute.yaml
-	kubectl delete -f examples/kubernetes/gateway.yaml
-	kubectl delete -f examples/kubernetes/gatewayclass.yaml
-	kubectl delete -f examples/kubernetes/httpbin.yaml
+	kubectl delete -f examples/kubernetes/quickstart.yaml
 
```

Comment:
Just to call it out: The order of these is changing. (The new order is `gatewayclass.yaml`, `gateway.yaml`, `httpbin.yaml`, `httproute.yaml`)

I don't think that matters, but I figured it should be said explicitly.